superset/config.py

```diff
@@ -258,6 +258,7 @@
 # method.
 # note: index option should not be overridden
 CSV_EXPORT = {"encoding": "utf-8"}
+CSV_EXPORT_USE_VERBOSE_NAMES_AS_HEADERS = True
```

Comment:
This should not default to `True`, as people may be depending on the current behaviour.